scripts/bucketize.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+set -o errexit -o pipefail
+
+# This script takes the built Hugo site and:
+#   * creates a new S3 bucket named with either the current PR number or Git SHA
+#   * creates a list of all Hugo-generated client-side ("meta-refresh") redirects that
+#     we'll use to produce proper 301s later.
+#   * pushes the content of the website into the new S3 bucket.
+#   * tests the built website, first for broken links, then with Cypress to ensure pages
+#     render and behave properly.
+#   * emits a metadata file containing information about the commit and bucket, which
+#     Pulumi will use to process its update.
+#   * writes a record to AWS Parameter Store relating the generated bucket to the commit
+#     responsible for producing it.
+#   * Posts a PR comment back to GitHub, if applicable.
+
+source ./scripts/common.sh
+
+# The docroot of the built website.
+build_dir="public"
+
+# The text file we'll write as an output result.
+metadata_file="$(origin_bucket_metadata_filepath)"
+
+# Verify we have at least 1000 index.html files in total across the site.
+if [ ! "$(find $build_dir -type f | grep index.html | wc -l)" -ge 1000 ]; then
+    echo "Page-count check failed. See the archive extraction log above for details."
```

Comment:
This error no longer applies outside the context of the archive job, right? The check is still useful, but I think we probably need to update this message.

Reply:
Ah yes, thanks -- the message doesn't make sense in the revised context. I'll update.